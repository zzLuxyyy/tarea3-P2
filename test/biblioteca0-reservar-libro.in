crearBiblioteca
crearSocio 12345678 Jim Dear 10 03 2004
agregarGeneroABiblioteca -1 1 Default
crearFecha 01/01/1966
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado en una historia de dolor y deseo" 1
agregarLibroBiblioteca
reservarLibroBiblioteca 12345678 431244557
imprimirReservasBiblioteca
agregarSocioBiblioteca
crearSocio 12345679 Raul Rodriguez 10 03 2001
reservarLibroBiblioteca 12345678 431244557
imprimirReservasBiblioteca
crearFecha 01/03/1970
prestarLibroBiblioteca 12345678 431244557
reservarLibroBiblioteca 12345677 431244557
imprimirReservasBiblioteca
reservarLibroBiblioteca 12345678 431244587
imprimirReservasBiblioteca
reservarLibroBiblioteca 12345678 431244557
reservarLibroBiblioteca 12345679 431244557
agregarSocioBiblioteca
imprimirReservasBiblioteca
reservarLibroBiblioteca 12345679 431244557
imprimirReservasBiblioteca
crearFecha 15/08/2025
devolverLibroBiblioteca 12345678 431244557 01/03/1970
crearSocio 12345677 Paola Berruti 10 02 2004
agregarSocioBiblioteca
reservarLibroBiblioteca 12345677 431244557
imprimirReservasBiblioteca
liberarBiblioteca
Fin