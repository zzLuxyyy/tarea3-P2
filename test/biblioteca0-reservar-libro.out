1>Biblioteca creada exitosamente.
2>Socio de ci 12345678 creado exitosamente.
3>Genero agregado a la biblioteca4>5>6>Libro de ISBN 431244557 agregado a la biblioteca.
7>Reserva del libro 431244557 para el socio 12345678 registrada.
8>Cola de Reservas:
9>Socio de CI 12345678 agregado a la biblioteca.
10>Socio de ci 12345679 creado exitosamente.
11>Reserva del libro 431244557 para el socio 12345678 registrada.
12>Cola de Reservas:
13>14>Préstamo del libro 431244557 al socio 12345678 registrado.
15>Reserva del libro 431244557 para el socio 12345677 registrada.
16>Cola de Reservas:
17>Reserva del libro 431244587 para el socio 12345678 registrada.
18>Cola de Reservas:
19>Reserva del libro 431244557 para el socio 12345678 registrada.
20>Reserva del libro 431244557 para el socio 12345679 registrada.
21>Socio de CI 12345679 agregado a la biblioteca.
22>Cola de Reservas:
Reserva de libro Lo bello y lo triste por Jim Dear.
23>Reserva del libro 431244557 para el socio 12345679 registrada.
24>Cola de Reservas:
Reserva de libro Lo bello y lo triste por Jim Dear.
Reserva de libro Lo bello y lo triste por Raul Rodriguez.
25>26>Devolución del libro 431244557 por el socio 12345678 registrada.
27>Socio de ci 12345677 creado exitosamente.
28>Socio de CI 12345677 agregado a la biblioteca.
29>Reserva del libro 431244557 para el socio 12345677 registrada.
30>Cola de Reservas:
Reserva de libro Lo bello y lo triste por Jim Dear.
Reserva de libro Lo bello y lo triste por Raul Rodriguez.
31>Biblioteca liberada exitosamente.
32>Fin.
